scripts/lib.sh

```diff
@@ -251,22 +273,86 @@ generate_dockerfile()
 
 	curlOptions=("-OL")
 	[ -n "${http_proxy:-}" ] && curlOptions+=("-x ${http_proxy:-}")
+
+	readonly dockerfile_template="Dockerfile.in"
 	readonly install_go="
 RUN cd /tmp ; curl ${curlOptions[@]} https://storage.googleapis.com/golang/go${GO_VERSION}.linux-${goarch}.tar.gz
 RUN tar -C /usr/ -xzf /tmp/go${GO_VERSION}.linux-${goarch}.tar.gz
 ENV GOROOT=/usr/go
 ENV PATH=\$PATH:\$GOROOT/bin:\$GOPATH/bin
 "
 
-	readonly dockerfile_template="Dockerfile.in"
+	# Rust agent
+	# rust installer should set path apropiately, just in case
+	local cmake_file="cmake-${CMAKE_VERSION}.tar.gz"
+	local cmake_dir="cmake-${CMAKE_VERSION}"
+	readonly install_cmake="
+RUN pushd /root; \
+    curl -sLO https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/${cmake_file}; \
```

Comment:
Indents seem to be off here and below as well.

Reply:
it is deliberately used here to identify RUN commands